dev/io.openliberty.microprofile.telemetry.internal.common/src/io/openliberty/microprofile/telemetry/api/OpenTelemetryInfo.java

```diff
@@ -1,31 +1,29 @@
 /*******************************************************************************
- * Copyright (c) 2023 IBM Corporation and others.
+ * Copyright (c) 2024 IBM Corporation and others.
  * All rights reserved. This program and the accompanying materials
  * are made available under the terms of the Eclipse Public License 2.0
  * which accompanies this distribution, and is available at
  * http://www.eclipse.org/legal/epl-2.0/
  *
  * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
  *******************************************************************************/
-package io.openliberty.microprofile.telemetry.internal.common.info;
+package io.openliberty.microprofile.telemetry.api;
 
 import io.opentelemetry.api.OpenTelemetry;
-import io.opentelemetry.api.trace.Tracer;
 
 public interface OpenTelemetryInfo {
+
     /**
-     * @return true if enabled
+     * @return true if the contained OpenTelemetry object was enabled at the time of creation, otherwise false.
      */
     public boolean getEnabled();
 
     /**
-     * @return the openTelemetry
+     * @return the held io.opentelemetry.api.OpenTelemetry object.
      */
     public OpenTelemetry getOpenTelemetry();
```

Comment:
Addendums to the previous comment about Javadoc:

- You should have at least one line of doc in addition to the `@return` tag, otherwise the summary for the method shows no data.
- We should use `@link` and `@code` tags appropriately when referring to other classes or methods.